.travis.yml

```diff
@@ -26,28 +24,35 @@ env:
   - FRAMEWORK=Zend2 TEST_REPO='-b 2.2 --recurse-submodules https://github.com/Naktibalda/codeception-zf2-tests' SUITES=functional
   - FRAMEWORK=ZendExpressive TEST_REPO='-b 2.2 --recurse-submodules https://github.com/Naktibalda/codeception-zend-expressive-tests' SUITES=functional
 matrix:
+  fast_finish: true
   include:
-  - php: 7.1
-    env: FRAMEWORK=Codeception SUITES=cli,unit,coverage TEST_PATH=. XDEBUG=1 PECL=mongodb
+  # Run our coverage suite on 7.2 (so it uses pcov)
+  - php: 7.2
+    env: FRAMEWORK=Codeception SUITES=cli,unit,coverage TEST_PATH=. PECL="mongodb" COVERAGE=1
   exclude:
-  - php: 7.1
-    env: FRAMEWORK=Codeception SUITES=cli,unit TEST_PATH=. XDEBUG=1 PECL=mongodb
+
+  - php: 5.6
+    env: FRAMEWORK=Yii2 TEST_REPO="https://github.com/Codeception/yii2-tests"
```

Comment:
What does this line do? I looked at the last build and Yii2 tests were executed on all versions of PHP.

Reply:
I'll try to remove it again; I had some issues with that specific combination when running tests.

Reply:
The advanced app template requires PHPUnit 6.5, which does not support PHP5.6.
So solutions are:
- We no longer test the advanced app template for any Codeception version.
- We no longer test the Yii2 module for PHP 5.6.
- We adapt the module test to dynamically include / exclude the advanced app template depending on PHP versions.

Reply:
https://github.com/yiisoft/yii2-app-advanced/issues/440